src/v/coproc/offset_storage_utils.cc

```diff
@@ -132,7 +70,7 @@ ss::future<ntp_context_cache> recover_offsets(
               key);
         } else {
             auto [itr, _] = recovered.emplace(
-              std::move(key), ss::make_lw_shared<ntp_context>(std::move(*log)));
+              key, ss::make_lw_shared<ntp_context>(std::move(*log)));
```

Comment:
what changed here? it doesn't seem like any of the reflection work would have affected this

Reply:
So nothing really changed here, the `std::move` wasn't ever moving to begin with because `key` is const as it is a key in a map and cannot be moved out of the map. Should i just leave it as is or put this in another commit?

Reply:
got it, thanks. I think its ok to leave it